std_msgs/Header header
flock_simulator/DuckieState[] duckie_states
flock_simulator/Request[] open_requests
flock_simulator/Request[] filled_requests
